tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -52,3 +54,25 @@ def test_upgrade():
     """
 
     run_ocs_upgrade()
+
+
+@ocs_upgrade
```

Comment:
this marker means that this test is an upgrade test. I think it should not be in use here. 
@AviadP @petr-balogh could you please review?

Reply:
changed to post_ocs_upgrade